style.css

```diff
@@ -0,0 +1,89 @@
+*{
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+    text-rendering: optimizeLegibility;
+    font-size: 20px;
+    transition: all 0.5;
+}
+nav{
+    display: flex;
+    justify-content: space-between;
+    padding: 5px 20px;
+    font-size: 30px;
+    align-items: center;
+}
+.na{
+    display: flex;
+    height: 10vh;
+}
+.search{
+    width: 60%;
+    justify-content: space-between;
+    padding-top: 20px;
+}
+.search div{
+    width: 98%;
+}
+.search div input{
+    width: 90%;
+    margin-right: 0%;
+}
+.search div i{
+    border: 1px solid;
+    margin-left: 0%;
+    padding: 1px;
+    border-bottom-right-radius: 10px;
+    border-top-right-radius: 10px;
+}
+
+
+.na img{
+    width: 120px;
+    height: auto;
+}
+.logo i{
+    margin-top: 20px;
+}
+.person{
+    padding-top: 20px;
+}
+.person a{
+text-decoration: none;
+margin-left: 20px;
+border: 1px solid;
+border-radius: 20%;
+color: green;
+padding-bottom: 0px;
+}
+.header{
+    padding: 50px;
+    display: flex;
+}
+.video{
+    flex-basis: 60%;
+    margin-right: 30px;
+}
+.video iframe{
+    width: 100%;
+}
+.video h3{
+    font-size: 15px;
+    color: rgba(0, 42, 255, 0.998);
+}
+.sidevids{
+    height: auto;
+    flex: 1;
+}
+.sidevids div{
+    display: flex;
+    flex-basis: 45%;
+    margin-top: 5px;
+}
+.sidevids div iframe{
+    width: 100%;
+    height: auto;
+}
+.sidevids div h1{
+    font-size: 15px;
+}
```

Comment:
Your code structure does not follow the standard format

Reply:
- [ ] Your comment says: `Your code structure does not follow the standard format`, okay fine 🤗, you gave your first review to the student, congrats @Divinity-dev  but..., let's assume you are the student and you receive this comment under the review, do you think you will understand what it means? in my opinion, am not sure at all, because it is a blur and not explanatory, as a reviewer, you need to make sure you have explained to the student `1. what` is wrong in his/her code and `2. why`  he/she should change them, you may talk about CSS best practices learned at microverse and explain to him it is important they follow them., in your case, you said the `what`, but you did not explain the `why`, kindly take an example to this(my review comment ) and make the same explanation for your review.

- [ ] `how to talk to the student`: when performing the review, please do not be imperative, we are not making orders but we make suggestions, in that case, the student will feel more respected and congrats for the work he/she has done up to here.

> Hint 👍 
> Congratulations @Divinity-dev for the awesome work you have done up to now, however, there are a few points we can work on to make it even better, your CSS code is fine but I believe we are missing some important structure regarding the Html and CSS best practice, referring to this (`link`), I would suggest you do so and so and so.... because ...so and so and so, what do you think about changing it?🥰